calculator_simple/src/main.cpp

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file
+ * @brief   GTK+3 (gtkmm, C++ Bindings) example Calculator app.
+ * 
+ *
+ * @author     Tommy Vegetables <leafyrabbet@gmail.com>
+ * @date       2020-05-16
+ * @copyright  2020, TenonGarden Productions
+ */
+#include "app_window.hpp"
+
+int main(int argc, char * argv[])
+{
+   auto app = Gtk::Application::create(
+      argc
+      , argv
+      , "dev.leafyrabbet.calculator.simple"
```

Comment:
I don't know if this reverse URI format is correct, or necessary. If not necessary, I'd suggest changing it to something more personally preferable and consistent, like the sub-project naming convention.

Reply:
https://developer.gnome.org/ChooseApplicationID/

Reply:
We don't actually own `leafyrabbet.dev` as a (top-level) domain, but this format is correct as suggested by the GNOME developer documentation, so I'm gonna consider this as a "non-issue" and resolve this.